src/main/java/com/salesforce/dataloader/mapping/LoadMapper.java

```diff
@@ -88,8 +92,8 @@ public void verifyMappingsAreValid() throws MappingInitializationException {
         for (Map.Entry<String, String> entry : getMappingWithUnmappedColumns(false).entrySet()) {
             String sfdcName = entry.getValue();
             if(StringUtils.hasText(sfdcName)) {
-                final Field f = getClient().getField(sfdcName);
-                if (f == null)
+                final HashSet<Field> fieldSet = getClient().getField(sfdcName);
+                if (fieldSet == null | fieldSet.isEmpty())
```

Comment:
The | was a typo.  I'll change that.  In the examples I tested it with 
it apparently didn't cause a problem.